charts/common/values.yaml

```diff
@@ -0,0 +1,600 @@
+      # How it works:
+      # 1. In the annotations of the pods a timestamp is set; When there is a new deployment the Kubernetes resource (e.g. Statefulset) will redeploy its pods because the timestamp has changed.
+      # 2. If the pullPolicy is set to 'Always' the image is always downloaded again and no cache ist used to get old image (useful when developing with imagetag latest).
+      forceRedeploy: false
+
+      # By default a component will be restarted during a deployment if the content of envSecret or envConfigMap changes.
+      # Normally we want that but there may be edge cases where we need the ability to disable this behavior (disableChecksumAnnotations: true).
+      disableChecksumAnnotations: false
+
+      # extraChecksumAnnotations let us define additional checksum annotations for custom content (eg. a ConfigMap) outside the common chart.
+      # key:          The key of the podAnnotation to store the checksum on.
+      # checksumFrom: The sha256 checksum is calculated from the file specified here. The directory "templates" in your custom helm chart is the ROOT dir.
+      #               So when you place your file in myChart/templates/app-a/config.yaml, use the example below:
+      extraChecksumAnnotations: []
+      # - key: checksum/myconfig
+      #   checksumFrom: /app-a/config.yaml
+      ## END ONLY FOR STATEFULSETS OR DEPLOYMENTS
+
+      # START ONLY FOR STATEFULSETS
+      # updateStrategy of statefulset. Can be "RollingUpdate" or "OnDelete". Comment out for using default ("RollingUpdate")
+      #updateStrategy: "OnDelete"
+
+      extraVolumeClaimTemplates: []
+        #- metadata:
+        #    name: foobar
+        #  spec:
+        #    accessModes: [ "ReadWriteOnce" ]
+        #    storageClassName: myStorageClass
+        #    resources:
+        #      requests:
+        #        storage: 2Gi
+      # END ONLY FOR STATEFULSETS
+
+      # START ONLY FOR DEPLOYMENTS
+      # strategy is a dictionary for specifying properties about the strategy on deployment updates
+      strategy: {}
+        # type of deployment. Can be "Recreate" or "RollingUpdate". Comment out for using default ("Recreate")
+        #type: "RollingUpdate"
+        # maxSurge is only needed when type is "RollingUpdate". The maximum number of pods that can be scheduled above the desired number of pods. Value can be an absolute number (ex: 5) or a percentage of desired pods (ex: 10%). Comment out for using default (1)
+        #maxSurge: 2
+        # maxUnavailable is only needed when type is "RollingUpdate". The maximum number of pods that can be unavailable during the update. Value can be an absolute number (ex: 5) or a percentage of desired pods (ex:10%). Comment out for using default (1)
+        #maxUnavailable: 50%
+      # END ONLY FOR DEPLOYMENTS
+
+      # START ONLY FOR JOBS OR CRONJOBS
+      # see https://v1-23.docs.kubernetes.io/docs/reference/kubernetes-api/workload-resources/job-v1/ for more details
```

Comment:
change to general link: https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/job-v1/

Reply:
done